javascript/javascript1/week4/voiceAssistant.js

```diff
@@ -0,0 +1,52 @@
+let nameHolder= "";
+const toDoList = [];
+const monthNames = ["January", "February", "March", "April", "May", "June",
+  "July", "August", "September", "October", "November", "December"
+];
+function getReply(command) {
+ 
+  // later down fields will be checked if includes some arithmetic statement. 
+  let fields = command.split('is');
+  
+    if (command === 'Hello my name is Benjamin' && nameHolder==="") {
+
+            const namePosition = command.search("is")+3;
+            nameHolder = command.substr(namePosition, command.length-1);
+            return "Nice to meet you Benjamin";
+  }
+  else if (command === 'Hello my name is Benjamin'){
+             return 'You already introduced to me!'
+  }
+  else if (command ==='What is my name') {
+      let xx= nameHolder ? nameHolder :`You haven't introduced to me!`;
+      return xx;
```

Comment:
Well done using a ternary operator! But the same thing about `return`. And also `xx` is not a good variable name.